databricks/koalas/series.py

```diff
@@ -485,9 +485,52 @@ def dropna(self, axis=0, inplace=False, **kwargs):
     def head(self, n=5):
         return _col(self.to_dataframe().head(n))
 
+    # TODO: Categorical type isn't supported (due to PySpark's limitation) and
+    # some doctests related with timestamps were not added.
     def unique(self):
-        # Pandas wants a series/array-like object
-        return _col(self.to_dataframe().unique())
+        """
+        Return unique values of Series object.
+
+        Uniques are returned in order of appearance. Hash table-based unique,
+        therefore does NOT sort.
+
+        .. note:: This method should only be used if the resulting Pandas object is expected
```

Comment:
we should return a koalas . Series, so . we don't have this issue.

Reply:
This returns a NumPy array in Pandas.

Reply:
yea my point is we shouldn't do this.

see "Return Koalas data structure for big data, and pandas data structure for small data" in https://github.com/databricks/koalas/pull/255/files